lib/ui/widgets/common/academy_local_image.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:filledstacked_academy/ui/common/app_colors.dart';
+import 'package:flutter/material.dart';
+import 'package:shimmer/shimmer.dart';
+
+class AcademyLocalImage extends StatelessWidget {
+  final String name;
+  final double height;
+  final double width;
+
+  const AcademyLocalImage(
+    this.name, {
+    super.key,
+    required this.height,
+    required this.width,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Stack(
```

Comment:
Instead of a stack can we use an the `CrossFade` widget. If that doesn't work, we can keep the Stack and do a cross fade with the Image and an empty SizedBox with width and height matching. 

I'd like it to fade in when the image is ready instead of popping in.

Reply:
What do you think now? 👇